src/main/java/org/folio/eusage/reports/api/EusageReportsApi.java

```diff
@@ -556,28 +556,49 @@ static String altKey(JsonObject jsonObject, String... keys) {
         });
   }
 
-  Future<JsonObject> lookupOrderLines(JsonArray poLinesAr, RoutingContext ctx) {
+  Future<JsonArray> lookupInvoiceLine(UUID poLineId, RoutingContext ctx) {
+    String uri = "/invoice-storage/invoice-lines?query=poLineId%3D%3D" + poLineId;
+    return createRequest(webClient, HttpMethod.GET, ctx, uri)
+        .send()
+        .compose(res -> {
+          if (res.statusCode() != 200) {
+            return Future.failedFuture("GET " + uri + " returned status code " + res.statusCode());
+          }
+          return Future.succeededFuture(res.bodyAsJsonArray());
```

Comment:
```suggestion
        .map(res -> {
          if (res.statusCode() != 200) {
            throw new NoStackTraceThrowable("GET " + uri + " returned status code " + res.statusCode());
          }
          return res.bodyAsJsonArray();
```
compose is for calling another async process, map is for non-async calculation.

Reply:
OK? Gives `unreported exception io.vertx.core.impl.NoStackTraceThrowable; must be caught or declared to be thrown`